cpp/binary/viewer/CMakeLists.txt

```diff
@@ -46,6 +47,15 @@ qt_add_executable(
     ${MODMESH_PYMOD_SPACETIME_SOURCES}
 )
 
+if(WIN32)
+    set_target_properties(
+        viewer
+        PROPERTIES
+        LINK_FLAGS_DEBUG "/SUBSYSTEM:CONSOLE" # open the console for debug mode
```

Comment:
If it opens an additional debug console in release build, it needs to be changed.

Reply:
in release mode, there is no cmd prompt.

Reply:
To make it apparent, I recommend to only add the switch when it's not in release mode.